app/src/main/res/values/strings.xml

```diff
@@ -7,10 +7,11 @@
     <string name="login_page_image_text">Login Page</string>
     <string name="forgot_password_login_view">Forgot Password?</string>
     <string name="new_here_register_button_on_login_view">New Here? Register!</string>
-    <string name="alreadyUser_button_on_register_view">New Here? Register!</string>
+    <string name="alreadyUser_button_on_register_view">New Here? Login!</string>
```

Comment:
If the user already exists then it doesn't make sense to say "New Here? Login", instead this should be "Already a member? Login" as per UX designs

Reply:
Updated!